zproject/urls.py

```diff
@@ -798,6 +798,11 @@
     path("policies/<slug:article>", policy_documentation_view),
 ]
 
+if not settings.CORPORATE_ENABLED:
```

Comment:
I added a comment referencing the `apps_view` comment here before merging, just to provide better discoverability of what's going on.